app/views/curation_concerns/base/_permission_form.html.erb

```diff
@@ -3,7 +3,7 @@
 <% public_perm = true if params[:controller] == 'batch' %>
 <% registered_perm = f.object.permissions.map { |perm| perm.access if perm.agent_name == "registered"}.compact.first %>
 
-<h2><% if params[:controller] == 'batch' %>Bulk <% end %>Permissions <% if params[:controller] == 'batch' %>
+<h2><% if params[:controller] == 'batch' %>Bulk <% end %><a name="permissions_display">Permissions </a><% if params[:controller] == 'batch' %>
```

Comment:
It is on <h2>: https://github.com/projecthydra/sufia/blob/master/app/views/curation_concerns/base/_permission_form.html.erb#L6